resources/usa_ammunition/server/sv_ammunition.lua

```diff
@@ -72,7 +72,7 @@ local WEPS_NO_MAGS = {
         AMMO_NAME = "12 Gauge Shells",
         MAX_CAPACITY = 8
     },
-    [GetHashKey("WEAPON_PUMPSHOTGUN_MK2")] = {
+    [[--[GetHashKey("WEAPON_PUMPSHOTGUN_MK2")] = {
```

Comment:
```suggestion
    [[--[[GetHashKey("WEAPON_PUMPSHOTGUN_MK2")] = {
```
isn't it supposed to be like that instead for a comment

Reply:
Yeah it looks weird on github but its because I commented out multiple lines. 

Also when I say "Old Weapons" I mean I just commented out the weapons that weren't replaced with custom models until I could get them replaced with custom models and metas.

Reply:
Also no the way I commented it out was correct.

Reply:
> "I just commented out the weapons that weren't replaced with custom models until I could get them replaced with custom models and metas."

And why not just leave them in ?

Reply:
We can. I'll change quickly